packages/tailwind/src/tailwind.tsx

```diff
@@ -175,16 +175,20 @@ export const Tailwind: React.FC<TailwindProps> = ({ children, config }) => {
       element,
       nonMediaQueryTailwindStylesPerClass,
     );
-
-    if (
-      element.type === "head" ||
-      (typeof element.type === "function" &&
-        "name" in element.type &&
-        element.type.name === "Head")
-    ) {
-      headElementIndex = i;
-    }
-  });
+  if (element.type === "head") {
+      headElementIndex = i; 
+  }
+  else if (
+    typeof element.type === "function" &&
```

Comment:
This seems to break Tailwind when using our [<Head>](https://github.com/resend/react-email/tree/canary/packages/head) component.